test/components/BarChart.test.js

```diff
@@ -1,3 +1,5 @@
+/* eslint no-undef: 0 */
```

Comment:
Ya, otherwise the test file is going to have linting complaints in the editor. We could add the whole `/test` directory to the ignore file?